src/Workspaces/Remote/ServiceHub/Services/DesignerAttributeDiscovery/RemoteDesignerAttributeDiscoveryService.cs

```diff
@@ -36,7 +36,7 @@ public ValueTask StartScanningForDesignerAttributesAsync(RemoteServiceCallbackId
                     analyzerProvider,
                     new IncrementalAnalyzerProviderMetadata(
                         nameof(RemoteDesignerAttributeIncrementalAnalyzerProvider),
-                        highPriorityForActiveFile: true,
+                        highPriorityForActiveFile: false,
```

Comment:
so this is correct, but still problematic :-/   we *do* want a system whereby a user touches a document, and we update the 'designable' state for it in VS asap.  Do we have any thoughts on how we might accomplish that moving forward?

Reply:
Diagnostic incremental analyzer handles this by running the solution crawler portion in **devenv.exe** and delegating the individual document/project analysis operations to OOP. For designer attribute discovery, we have a few options:

1. Rely on Solution Crawler to minimize the number of document analysis operations that are requested following a document change. Analysis operations would directly prioritize open documents, but open documents would scan quickly following changes by nature of work being added to a short queue.
2. Move designer attribute solution crawling to **devenv.exe**, and delegate to OOP similar to diagnostics.
3. Leave this analyzer unchanged (essentially option 1), but add a parallel API that can perform targeted updates of a document following context switches.